src/ConfirmationTooltip/index.tsx

```diff
@@ -0,0 +1,78 @@
+import "../../node_modules/tippy.js/animations/shift-away.css";
+import React from "react";
+import Tippy from "@tippy.js/react";
+import { TippyStyles } from "../shared/TippyStyles";
+import { useSpaceKitProvider } from "../SpaceKitProvider";
+
+type TippyProps = React.ComponentProps<typeof Tippy>;
+
+interface Props
+  extends Pick<TippyProps, "children" | "content">,
+    Partial<Pick<TippyProps, "placement">> {
+  /**
+   * Force the tooltip to be visible. Only use this for testing purposes; don't
+   * use this to programatically control the component
+   */
+  forceVisibleForTestingOnly?: boolean;
+}
+
+/**
+ * Display a non-interactive tooltip after an element is clicked on and then
+ * remove the tooltip after a short period of time
+ */
+export const ConfirmationTooltip: React.FC<Props> = ({
+  children,
+  forceVisibleForTestingOnly = false,
+  ...props
+}) => {
+  const { disableAnimations } = useSpaceKitProvider();
+
+  /**
+   * Store result of `useTimeout` so we can cancel the timeout if this component
+   * is unmounted before the timeout fires
+   */
+  const timeoutRef = React.useRef<number | null>(null);
+
+  // When this component is being unmounted, remove the timeout if one exists.
+  // This will prevent trying to call methods on destroyed components
+  React.useEffect(() => {
+    return () => {
+      if (timeoutRef.current) {
+        window.clearTimeout(timeoutRef.current);
+      }
+    };
+  });
+
+  return (
+    <>
+      <TippyStyles />
+      <Tippy
+        arrow={false}
+        animation={!disableAnimations ? "shift-away" : undefined}
+        hideOnClick={forceVisibleForTestingOnly ? false : undefined}
+        theme="space-kit"
+        trigger={forceVisibleForTestingOnly ? "manual" : "click"}
+        visible={forceVisibleForTestingOnly ? true : undefined}
+        {...props}
+        onShow={instance => {
+          if (timeoutRef.current) {
+            window.clearTimeout(timeoutRef.current);
+          }
+
+          timeoutRef.current = window.setTimeout(() => {
+            instance.hide();
+          }, 3000);
```

Comment:
Seems like the delay time should be a prop?  Or "short | medium | long".  But we can also wait for a use-case.

Reply:
I wanted to explicitly avoid configuration to guarantee consistency. I'll make sure to review this value with @adamatorres before merging. Even if we change our mine; it should be easy to quickly iterate on this.